model_demos/cv_demos/dla/onnx_dla.py

```diff
@@ -0,0 +1,78 @@
+import os
+import urllib
+
+import onnx
+import pybuda
+import requests
+import torchvision.transforms as transforms
+from PIL import Image
+from pybuda._C.backend_api import BackendDevice
+
+
+def run_dla_onnx(variant):
+    # Load model function
+    model_name = f"dla_{variant}_pytorch"
+
+    # Set PyBuda configuration parameters
+    compiler_cfg = pybuda.config._get_global_compiler_config()
+    compiler_cfg.balancer_policy = "Ribbon"
+    compiler_cfg.default_df_override = pybuda._C.Float16_b
+    os.environ["PYBUDA_RIBBON2"] = "1"
+
+    available_devices = pybuda.detect_available_devices()
+    if available_devices:
+        arch = available_devices[0]
+        if arch == BackendDevice.Grayskull:
+            if variant == "dla102x2":
+                os.environ["PYBUDA_FORCE_CONV_MULTI_OP_FRACTURE"] = "1"
+
+    # Load data sample
+    url = "https://images.rawpixel.com/image_1300/cHJpdmF0ZS9sci9pbWFnZXMvd2Vic2l0ZS8yMDIyLTA1L3BkMTA2LTA0Ny1jaGltXzEuanBn.jpg"
+    image = Image.open(requests.get(url, stream=True).raw)
+    label = "tiger"
+
+    # Preprocessing
+    transform = transforms.Compose(
+        [
+            transforms.Resize(256),
+            transforms.CenterCrop(224),
+            transforms.ToTensor(),
+            transforms.Normalize(mean=[0.485, 0.456, 0.406], std=[0.229, 0.224, 0.225]),
+        ]
+    )
+    img_tensor = transform(image).unsqueeze(0)
+
+    # Download Model
+    onnx_model_path = f"dla/{variant}_Opset18.onnx"
```

Comment:
Can you add cleanup logic to delete those ONNX downloaded files at the end of the script?

Reply:
Sure @nvukobratTT . I will update this patch accordingly.

Reply:
I have added the cleanup code now. Please take a look @nvukobratTT